GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/OldLionsCourt.cs

```diff
@@ -279,5 +280,39 @@ protected override void SetInstanceBuffs(ParsedEvtcLog log)
                 }
             }
         }
+
+        internal override void ComputePlayerCombatReplayActors(AbstractPlayer p, ParsedEvtcLog log, CombatReplay replay)
+        {
+            // Fixation
+            IEnumerable<AbstractBuffEvent> fixatedVermillion = log.CombatData.GetBuffData(FixatedOldLionsCourt)
```

Comment:
This would look cleaner with:
fixated = log.CombatData.GetBuffData(FixatedOldLionsCourt).Where(buff => buff.To == p.AgentItem);
Use the fixated enumerable for each sub type.